ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/tunnel-interfaces.j2

```diff
@@ -64,6 +70,9 @@ interface {{ tunnel_interface.name }}
 {%     if tunnel_interface.path_mtu_discovery is arista.avd.defined(true) %}
    tunnel path-mtu-discovery
 {%     endif %}
+{%     if tunnel_interface.ipsec_profile is arista.avd.defined and tunnel_interface.tunnel_mode == "ipsec" %}
```

Comment:
We should not have logic in eos_cli_config_gen. Just configure what is given as long as it is accepted by EOS.
```suggestion
{%     if tunnel_interface.ipsec_profile is arista.avd.defined %}
```